src/main/java/com/google/sps/utils/EmailUtils.java

```diff
@@ -14,18 +14,37 @@
 
 package com.google.sps.utils;
 
+// using SendGrid's Java Library
+// https://github.com/sendgrid/sendgrid-java
+import com.sendgrid.Method;
+import com.sendgrid.Request;
 import com.sendgrid.Response;
-import com.sendgrid.helpers.mail.objects.Content;
-import com.sendgrid.helpers.mail.objects.Email;
+import com.sendgrid.SendGrid;
 import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.stream.Stream;
+import java.util.Map;
 import java.util.HashMap;
+import java.lang.ClassLoader;
+import java.net.URI;
+import java.net.URL;
+import java.net.URISyntaxException;
 
-/** EmailUtils includes the basic methods used when formatting emails. */
-public interface EmailUtils {
+// Used to format contents of an email
+public class EmailUtils {
 
   // Returns the contents of the file specified at filePath as a String. Useful for converting
   // predefined email templates to text.
-  public String fileContentToString(String filePath) throws IOException;
+  public static String fileContentToString(String fileName) throws IOException {
+    StringBuilder contentBuilder = new StringBuilder();
+    try (Stream<String> stream =
+        Files.lines(Paths.get(getEmailTemplateResource(fileName)), StandardCharsets.UTF_8)) {
+      stream.forEach(s -> contentBuilder.append(s).append("\n"));
+    }
```

Comment:
I don't think having this try does anything without a catch because the error just gets thrown

Reply:
l think it ensures the stream gets closed.

Reply:
l think it ensures the stream gets closed.

Reply:
Done